util/dump_index.py

```diff
@@ -9,7 +9,7 @@
 - Run:
 
 virtualenv ~/virtualenv/dump_index
-source ~/virtualenv/dump_index/bin/activate
+source ~/virtualenv/elasticsearch/bin/dump_index
```

Comment:
Revert, the original is correct

Reply:
Ugh not sure why this keeps reverting, done.